### PR TITLE
Adding shelephant and mergedeep

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tdegeus** [comment]
@clarketm since you wrote mergedeep, are you interested to be a co-maintainer of mergedeep's feedstock ?

**Comment by tdegeus** [comment]
Ready for review @conda-forge/help-python

File: recipes/mergedeep/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/clarketm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (synapticarbors) [inline recipes/mergedeep/meta.yaml:16]: Can you use the canonical form of this from the example:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35

File: recipes/mergedeep/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - pip
+  run:
+    - python >=3
+
+test:
+  imports:
+    - mergedeep
+
+about:
+  home: https://github.com/clarketm/{{ name }}
```
Comment (synapticarbors) [inline recipes/mergedeep/meta.yaml:30]: Just for ease of copying and pasting, could you not use jinja template variables in the `about` section for urls.

File: recipes/mergedeep/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
```
Comment (synapticarbors) [inline recipes/mergedeep/meta.yaml:20]: `python >=3.6` here under `host` and under `run` in keeping with the upstream's specification of `python_requires` in the `setup.py`

File: recipes/shelephant/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (synapticarbors) [inline recipes/shelephant/meta.yaml:16]: same comment with the `script` as in the other recipe.

File: recipes/shelephant/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    - pyyaml
+    - numpy
+
+test:
+  imports:
+    - shelephant
+  commands:
+    - shelephant_dump --help
+
+about:
```
Comment (synapticarbors) [inline recipes/shelephant/meta.yaml:42]: same comment about the jinja variables as in the other recipe.

File: recipes/shelephant/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    - shelephant_rm = shelephant.cli.shelephant_rm:main
+    - shelephant_send = shelephant.cli.shelephant_send:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - docopt
```
Comment (synapticarbors) [inline recipes/shelephant/meta.yaml:31]: Use the version restrictions in the upstream `setup.py`:

https://github.com/tdegeus/shelephant/blob/main/setup.py#L21

File: recipes/shelephant/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    - click
+    - pyyaml
+    - numpy
+
+test:
+  imports:
+    - shelephant
+  commands:
+    - shelephant_dump --help
```
Comment (synapticarbors) [inline recipes/shelephant/meta.yaml:40]: Add a `commands` test for each entry point if possible

**Comment by tdegeus** [comment]
Thanks @synapticarbors for you thorough review!

**Comment by synapticarbors** [comment]
I'm happy to merge this once the CI is finished. If you want to wait on the response from the possible co-maintainer, let me know. Otherwise I'll just merge and you can potentially add them in the feedstock.

**Comment by tdegeus** [comment]
Let's merge now and add the possible co-maintainer later!

**Pull request merged**